client/webserver/site/src/js/markets.ts

```diff
@@ -598,7 +598,8 @@ export default class MarketsPage extends BasePage {
 
   /* setHighLow calculates the high and low rates over the last 24 hours. */
   setHighLow () {
-    const cache = this.market?.candleCaches[fiveMinBinKey]
+    if (!this.market) return
+    const cache = this.market.candleCaches[fiveMinBinKey]
```

Comment:
Unnecessary. The only place `setHighLow` is called is in `handleCandlesRoute`, where we already check for `this.market`.